src/main/java/graphql/language/DirectiveDefinition.java

```diff
@@ -35,7 +35,7 @@ protected DirectiveDefinition(String name,
                                   List<Comment> comments,
                                   IgnoredChars ignoredChars,
                                   Map<String, String> additionalData) {
-        super(sourceLocation, comments, ignoredChars, additionalData);
+        super(sourceLocation, comments, ignoredChars, additionalData, description);
         this.name = name;
         this.description = description;
```

Comment:
the super class `AbstractDescribedNode` does contain the description, so no derived class should contain description (otherwise it is "overridden" in a non obvious way)

Reply:
Should be okay now, I removed instances of description from derived classes.